apitest/src/main/resources/resident/GetRemainingupdatecountbyIndividualIdRequest/GetRemainingupdatecountbyIndividualIdRequest.yml

```diff
@@ -52,7 +55,8 @@ GetRemainingupdatecountbyIndividualIdRequest:
 
    Resident_GetRemainingupdatecountbyIndividualIdRequest_Invalid_Token:
       endPoint: /idrepository/v1/identity/{individualId}/update-counts?idType={idType}&attribute_list={attribute_list}
-      role: partner
```

Comment:
Role should not be changed here

Reply:
@mohanachandran-s Role is Resident only,